solr/core/src/test/org/apache/solr/update/RootFieldTest.java

```diff
@@ -54,7 +54,7 @@ public static void beforeTest() throws Exception {
     solrClientTestRule.startSolr(Path.of(SolrTestCaseJ4.TEST_HOME()));
 
     solrClientTestRule
-        .newCollection(DEFAULT_TEST_COLLECTION_NAME)
```

Comment:
are we going away from using `DEFAULT_TEST_COLLECTION_NAME` everywhere?   I'm okay with that, just want to do it everywhere then ;-)

Reply:
That was a misunderstanding. We aren't going away from using going away from using `DEFAULT_TEST_COLLECTION_NAME`